src/components/buttons/EdgeSwitch.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import { TouchableOpacity, View } from 'react-native'
+
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+
+interface EdgeSwitchProps {
+  labelA: string
+  labelB: string
+  onSelectA: () => void
+  onSelectB: () => void
+}
+
+const EdgeSwitch: React.FC<EdgeSwitchProps> = ({ labelA, labelB, onSelectA, onSelectB }) => {
```

Comment:
I somewhat prefer the `props => { const { ... } = props` style, where the destructuring is its own statement. It's not a big difference at this scale, but it becomes meaningful for bigger / more complicated components like our text inputs. Even though `EdgeSwitch` is small enough to get away with inline destructuring, it's better to keep our components consistent with each other.

Reply:
Came from an AI prompted change, my bad